src/Server/cacheClient.tsx

```diff
@@ -82,27 +95,14 @@ const redisCacheSetup = cb => {
   }
 
   const cacheGet = async key => {
-    return await safeCacheCommand("get", key)
+    return await cacheCommandWithTimeout("get", key)
   }
 
```

Comment:
how about the `onComplete`, when do we call it?

Reply:
Ah, you know we don't any more. That was for interfacing with the old (unnecessary) cache adaptor that was deleted in this PR. Will update